apps/nextjs/src/app/api/trpc/[trpc]/route.ts

```diff
@@ -3,7 +3,8 @@ import { fetchRequestHandler } from "@trpc/server/adapters/fetch";
 import { appRouter, createTRPCContext } from "@acme/api";
 import { auth } from "@acme/auth";
 
-export const runtime = "edge";
+// Uncomment this line to enable edge runtime if your environment supports it
+// export const runtime = "edge";
```

Comment:
```suggestion
```

i'd rather just remove it. people who need and want it will know how to enable it